bower.json

```diff
@@ -29,7 +30,7 @@
     "type"
   ],
   "dependencies": {
-    "ramda": "^0.14.0"
+    "ramda": "0.1.x"
```

Comment:
Ah, my bad. So you corrected this already yes? I'm a tad unfamiliar with the whole interface here.